.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "/home/loki/code/go-cli/main.go",
+            "args": [
+                "update", "volume", "move", "pvc-6d531e0e-fc5a-4b35-b752-bf75fc5dc690", "66eb21cb-e78a-4d8a-989e-3df3913f2c3e", "d4e910ac-394b-4fdb-be51-27822c15b682"
+            ],
+        }
+    ]
+}
```

Comment:
Did you mean to commit this? If so we should change `/home/loki/code/go-cli/` to `${workspaceFolder}`

Reply:
I didn't mean to, I fell victim to the `git add .`